tests/core/test_dynamic_schemas.py

```diff
@@ -0,0 +1,43 @@
+import copy
+import time
+import pytest
+
+
+@pytest.mark.nonparallel
+def test_dynamic_schemas_update(request, admin_mc):
+    assert not schema_has_field(admin_mc)
+
+    eks_schema = admin_mc.client.by_id_dynamicSchema(
+        'amazonelasticcontainerserviceconfig')
+
+    new_field = copy.deepcopy(eks_schema.resourceFields['displayName'])
+    new_field.description = 'My special field.'
+    setattr(eks_schema.resourceFields, 'mySpecialField', new_field)
+
+    admin_mc.client.update_by_id_dynamicSchema(eks_schema.id, eks_schema)
+    request.addfinalizer(lambda: cleanup_extra_field(admin_mc))
+    assert schema_has_field(admin_mc)
+
+
+def cleanup_extra_field(admin_mc):
+    eks_schema = admin_mc.client.by_id_dynamicSchema(
+        'amazonelasticcontainerserviceconfig')
+    delattr(eks_schema.resourceFields, 'mySpecialField')
+    admin_mc.client.delete(eks_schema)
+    admin_mc.client.create_dynamicSchema(eks_schema)
+    time.sleep(1)
+    assert not schema_has_field(admin_mc)
```

Comment:
Probably shouldn't assert in a finalizer

Reply:
Removed assert.